first_hour_on_vwb/working_with_resources.ipynb

```diff
@@ -0,0 +1,1436 @@
+    "            description='Add resource',\n",
+    "            disabled=False,\n",
+    "            button_style='',\n",
+    "            tooltip='Click to add a git repository',\n",
+    "            icon='plus'\n",
+    "        )\n",
+    "        self.add_repo_button.on_click(self.add_repo)\n",
+    "        self.vb = widgets.VBox(\n",
+    "            [\n",
+    "                self.label,\n",
+    "                self.cloning_drop_down,\n",
+    "                self.input_name,\n",
+    "                self.input_description,\n",
+    "                self.input_repo_url,\n",
+    "                self.add_repo_button,\n",
+    "                self.output\n",
+    "            ],\n",
+    "            layout=vbox_layout\n",
+    "        )\n",
+    "\n",
+    "    def add_repo(self, b):\n",
+    "        with self.output:\n",
+    "            print(self.cloning_drop_down.value)\n",
+    "            # Cast description input to string.\n",
+    "            description_content = f\"\\\"{self.input_description.value}\\\"\"\n",
+    "            print(\"Running command to add repo...\")\n",
+    "            terraCommand = f\"\"\"terra resource add-ref git-repo \\\\\n",
+    "            --cloning={self.cloning_drop_down.value} \\\\\n",
+    "            --description={description_content} \\\\\n",
+    "            --name={self.input_name.value} \\\\\n",
+    "            --repo-url={self.input_repo_url.value} \\\\\n",
+    "            --workspace={CURRENT_WORKSPACE_ID}\"\"\"\n",
+    "            print(terraCommand)\n",
+    "            result = subprocess.run([\"terra\", \"resource\", \"add-ref\", \"git-repo\",\n",
+    "                                     f\"--cloning={self.cloning_drop_down.value}\",\n",
+    "                                     f\"--description={description_content}\",\n",
+    "                                     f\"--name={self.input_name.value}\",\n",
+    "                                     f\"--repo-url={self.input_repo_url.value}\",\n",
+    "                                     f\"--workspace={CURRENT_WORKSPACE_ID}\"],\n",
+    "                                    capture_output=True,\n",
+    "                                    text=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "\n",
+    "\n",
+    "# Instantiate widget\n",
+    "add_repo_widget = AddRepoWidget()\n",
+    "display(add_repo_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2. Add a BigQuery dataset\n",
+    "\n",
+    "Run the cell below to create a widget that adds a BigQuery dataset as a referenced resource to your workspace.\n",
+    "\n",
+    "Then, populate the input fields:\n",
+    "\n",
+    "- `Cloning`: Read about these options [in the Cloning Instruction section](#choosing-the-right-cloning-instruction).\n",
+    "- `Project ID`: The Google Cloud Project ID of your dataset.\n",
```

Comment:
The UI calls it `GCP Project ID`, lets have it match here.

Reply:
fixed!